### PR TITLE
x/net CVE

### PR DESCRIPTION
**Pull request by Bslabe123**
# Description

Fix for CVE-2021-44716

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -130,6 +130,7 @@ replace (
 	// Required for proper serialization of CRDs
 	github.com/renstrom/dedent => github.com/lithammer/dedent v1.0.0
 	golang.org/x/net v0.0.0-20211112202133-69e39bad7dc2 => golang.org/x/net v0.0.0-20211209124913-491a49abca63
+	golang.org/x/net v0.0.0-20211205041911-012df41ee64c => golang.org/x/net v0.0.0-20211209124913-491a49abca63
```
Comment (jackstine) [inline go.mod:133]: I am curious as to why there are two replaces for `golang.org/x/net`. Does this mean that there are 2 modules using different versions of this package?

Comment (Bslabe123) [reply go.mod:133]: Yep, Went with a targeted replace instead of a mass replace to make the change as small as possible

**Review by jackstine — APPROVED** [review]
LGTM

**Pull request merged**